package.json

```diff
@@ -4,21 +4,24 @@
   "version": "0.4.0",
   "license": "MIT",
   "repository": "https://github.com/bigtestjs/interactor",
-  "main": "dist/index.js",
-  "module": "src/index.js",
+  "module": "dist/index.js",
+  "files": [
+    "dist",
+    "docs",
+    "src"
```

Comment:
So `src` is not _technically_ needed. But I've definitely found it useful in the past when a package includes the source in `node_modules`.